src/double_arithmetic.jl

```diff
@@ -295,6 +295,24 @@ end
     y
 end
 
+"""
+    double_cube(x::Float64)
+Convert `x` to a DoubleFloat64 and then compute `x*x*x`.
+"""
+@inline function double_cube(a::Float64)
+    hi, lo = two_cube(a)
+    return FastDouble(hi, lo)
+end
+
+"""
+    cube(x::DoubleFloat64)
+Compute `x * x * x` in a more efficient way.
+"""
+@inline function cube(a::DoubleFloat64)
+    hi, lo = two_cube(a.hi)
+    return FastDouble(hi, lo)
```

Comment:
We loose half of the precision here! I think the way to go is similar to the `square(a::DoubleFloat64)` function, i.e.
consider the trinomial expansion of (hi+lo)^3 (in square it is the binomial expansion (hi+lo)^2)

Reply:
hard to believe I overlooked that --- thanks for being on the ball

Reply:
yes -- I will do both tests for two_cube and cube
and to that end, as I need to pull up some older maple code, I am closing this for now

I went at QD two or three years ago.  Julia was less stable and not quite as facile then. 
I am glad that you have done this work -- thanks.